control-plane/csi-driver/node/src/main.rs

```diff
@@ -224,6 +208,17 @@ impl CsiServer {
             let uds = UnixListener::bind(csi_socket).unwrap();
             info!("CSI plugin bound to {}", csi_socket);
 
+            // Change permissions on CSI socket to allow non-privileged clients to access it
+            // to simplify testing.
+            if let Err(e) = fs::set_permissions(
```

Comment:
Question: Why do we need to change the permission? How was it working without it?

Reply:
To make testing easier, otherwise we can't connect to the socket without "chmoding" it.

Reply:
Okay.